src/Distributions.jl

```diff
@@ -230,6 +234,7 @@ export
     rate,               # get the rate parameter
     sampler,            # create a Sampler object for efficient samples
     scale,              # get the scale parameter
+    scale!,             # provide storage for the scale parameter (used in multivariate distribution mvlognormal)
```

Comment:
@kmsquire @StefanKarpinski True, but it's an occasion where Julia's out-of-class function definitions show a problem of - excuse the pun - scalability. "scale" has a different meaning for matrices as it has for distributions. In a C++ class for the lognormal distirbution, we wouldn't think twice about using the function name (even if the standard library would contain an entirely unrelated function "scale") because the class context would define the semantics of the word, 

I have to say that one thing that surprised me about Julia is that if a method like "scale" is defined in base, and I want to also define it in my module, I have to import and then export it again to avoid warnings (which happened the last time I tried this in 0.3 - haven't tried this anymore in 0.4 so apologies if already changed). A more scalable behaviour, in my view, would be to allow separate definition in the two namespaces, and if I import the two definitions into the same workspace, let the mulitple-dispatch figure out which one to call.